bedwars-plugin/src/main/java/com/andrei1058/bedwars/language/SimplifiedChinese.java

```diff
@@ -0,0 +1,505 @@
+        addContentMessages(yml, "tnt", ConfigPath.SHOP_PATH_CATEGORY_UTILITY, "{color}TNT", Arrays.asList("&7花费：{cost} {currency}", "", "&7立即点燃, 适合炸点东西", "", "{quick_buy}", "{buy_status}"));
+        addContentMessages(yml, "ender-pearl", ConfigPath.SHOP_PATH_CATEGORY_UTILITY, "{color}末影珍珠", Arrays.asList("&7花费：{cost} {currency}", "", "&7偷家最快的方式", "", "{quick_buy}", "{buy_status}"));
+        addContentMessages(yml, "water-bucket", ConfigPath.SHOP_PATH_CATEGORY_UTILITY, "{color}水桶", Arrays.asList("&7花费：{cost} {currency}", "", "&7减缓敌人的速度", "&7也可以防止TNT破坏方块", "", "{quick_buy}", "{buy_status}"));
+        addContentMessages(yml, "bridge-egg", ConfigPath.SHOP_PATH_CATEGORY_UTILITY, "{color}搭桥蛋", Arrays.asList("&7花费：{cost} {currency}", "", "&7丢出后在其轨迹形成一座桥", "", "{quick_buy}", "{buy_status}"));
+        addContentMessages(yml, "magic-milk", ConfigPath.SHOP_PATH_CATEGORY_UTILITY, "{color}魔法牛奶", Arrays.asList("&7花费：{cost} {currency}", "", "&7喝下后60秒内不会触发陷阱", "", "{quick_buy}", "{buy_status}"));
+        addContentMessages(yml, "sponge", ConfigPath.SHOP_PATH_CATEGORY_UTILITY, "{color}海绵", Arrays.asList("&7花费：{cost} {currency}", "", "&7用来吸水不错", "", "{quick_buy}", "{buy_status}"));
+        addContentMessages(yml, "Compact Pop-up Tower", ConfigPath.SHOP_PATH_CATEGORY_UTILITY, "{color}袖珍弹出塔", Arrays.asList("&花费: {cost} {currency}", "", "&7放置一个袖珍弹出塔", "&7塔防！", "", "{quick_buy}", "{buy_status}"));
+
+        yml.addDefault(Messages.UPGRADES_MENU_GUI_NAME_PATH + "default", "团队升级和陷阱");
+        yml.addDefault(Messages.MEANING_NO_TRAP, "无陷阱！");
+        yml.addDefault(Messages.FORMAT_UPGRADE_TRAP_COST, "&7花费：{currencyColor}{cost} {currency}");
+        yml.addDefault(Messages.FORMAT_UPGRADE_COLOR_CAN_AFFORD, "&e");
+        yml.addDefault(Messages.FORMAT_UPGRADE_COLOR_CANT_AFFORD, "&c");
+        yml.addDefault(Messages.FORMAT_UPGRADE_COLOR_UNLOCKED, "&a");
+        yml.addDefault(Messages.FORMAT_UPGRADE_TIER_LOCKED, "&7");
+        yml.addDefault(Messages.FORMAT_UPGRADE_TIER_UNLOCKED, "&a");
+        yml.addDefault(Messages.UPGRADES_LORE_REPLACEMENT_CLICK_TO_BUY, "{color}点击购买！");
+        yml.addDefault(Messages.UPGRADES_LORE_REPLACEMENT_INSUFFICIENT_MONEY, "{color}你没有足够的{currency}！");
+        yml.addDefault(Messages.UPGRADES_LORE_REPLACEMENT_LOCKED, "&c已锁定！");
+        yml.addDefault(Messages.UPGRADES_LORE_REPLACEMENT_UNLOCKED, "{color}已解锁！");
+        yml.addDefault(Messages.UPGRADES_UPGRADE_BOUGHT_CHAT, "&a{player}购买了&6{upgradeName}");
+        yml.addDefault(Messages.UPGRADES_UPGRADE_TIER_ITEM_NAME.replace("{name}", "forge").replace("{tier}", "tier-1"), "{color}铁锭熔炉");
+        yml.addDefault(Messages.UPGRADES_UPGRADE_TIER_ITEM_LORE.replace("{name}", "forge").replace("{tier}", "tier-1"), Arrays.asList("&7增加岛上资源的生成速度", "", "{tierColor}等级 1：+50%生成速率, &b{cost} {currency}", "&7等级 2：+100%生成速率, &b8钻石", "&7等级 3：生成绿宝石, &b12钻石", "&7等级 4：+200%生成速率, &b16钻石", ""));
```

Comment:
In the latest version there are no tiers anymore. All the upgrade lores are based on new placeholders, are you able to remove the other tiers and correct them with the right placeholders? if you need an example just look at the english version in the master branch